ngmix/priors/priors.py

```diff
@@ -59,11 +59,10 @@ class FlatPrior(PriorBase):
         The minimum value of the allowed range.
     maxval: float
         The maximum value of the allowed range.
-    rng: np.random.RandomState or None
-        An RNG to use. If None, a new RNG is made using the numpy global RNG
-        to generate a seed.
+    rng: np.random.RandomState
+        An random number generator (RNG) to use.
     """
-    def __init__(self, minval, maxval, rng=None):
+    def __init__(self, minval, maxval, rng):
```

Comment:
```suggestion
    def __init__(self, minval, maxval, *, rng):
```

Do we want to use something like this to preserve the API?

Reply:
We may not want to. Asking a question.

Reply:
I don't think that preserves the api does it?

Also with the way I did it one can always send the args as keyword arguments, python allows that. They can even be out of order.   e.g. this works as expected

```python
def go(x, y, z):
    print(x, y, z)

go(z=0.1, y=2.0, x=100)
```
So I think it is effectively the same either way.  So I'm not sure which is preferable or even effectively different.  What do you think?

Reply:
Ah right. Well the change allows people to pass the rng without specifying it as a kwarg. That is fine and you right that it is backwards compat.